docs/_toc.yml

```diff
@@ -136,7 +136,22 @@ subtrees:
   - file: developers/index
     subtrees:
     - entries:
-      - file: developers/contributing
+      - file: developers/contributing/index
+        subtrees:
+        - entries:
+          - file: developers/contributing/dev_install
+          - file: developers/contributing/testing
+          - file: developers/contributing/translations
+          - file: developers/contributing/performance/index
+            subtrees:
+            - entries:
+              - file: developers/profiling
+              - file: developers/benchmarks
```

Comment:
```suggestion
              - file: developers/contributing/performance/profiling
              - file: developers/contributing/performance/benchmarks
```
Nothing shows up in the subtree in the sidebar for Performance:
<img width="232" alt="image" src="https://github.com/napari/docs/assets/76622105/196a8ec0-788b-47af-ab8d-b633c1d3ef3c">

Reply:
Fixed!